src/core/swap.c

```diff
@@ -827,7 +827,7 @@ static void swap_enter_activating(Swap *s) {
                 }
         }
 
-        r = exec_command_set(s->control_command, "/sbin/swapon", NULL);
+        r = exec_command_set(s->control_command, "/sbin/swapon", "-f", NULL);
```

Comment:
Please write this as `--fixpgsz` for avoidance of doubt.

Reply:
That was actually my first choice. I only used short option because systemd seems to prefer that (judging from other calls of `exec_command_set` and `exec_command_append`).